css-two-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+p,
+h3,
+h1 {
+  font-family: Karla, sans-serif;
+}
+
+h1 {
+  text-transform: uppercase;
+  font-size: 63px;
+  margin-top: 100px;
+  margin-bottom: 22px;
+  letter-spacing: 0.25rem;
+}
+
+p {
+  line-height: 1.75em;
+  font-size: 1.15em;
+  color: #5f5f5f;
+}
+
+.header-column-full > p {
+  font-size: 1.2em;
+}
+
+h3 {
+  text-transform: uppercase;
+  letter-spacing: 0.1rem;
+}
+
+.header-column-full {
+  text-align: center;
+}
```

Comment:
Again, this should be named `text-center` or something to that effect. `column-` classes only include `flex-basis` or `width`, anything else is not styling to apply to a `column-` class